packages/salesforcedx-vscode-core/src/util/orgUtil.ts

```diff
@@ -27,6 +30,20 @@ export async function setUpOrgExpirationWatcher(orgList: OrgList) {
   */
 }
 
+export async function getAuthFieldsFor(username: string): Promise<AuthFields> {
```

Comment:
@ken, I need to consolidate these two functions wit what's in OrgList.ts.  I'll probable move them out of OrgList and move into a utils file.

Reply:
Sounds good - utils/authUtil?

Reply:
yep